ios/Classes/SwiftFlutterBarcodeScannerPlugin.swift

```diff
@@ -111,13 +111,24 @@ public class SwiftFlutterBarcodeScannerPlugin: NSObject, FlutterPlugin, ScanBarc
                                 
                             }
                         } else {
-                            let alert = UIAlertController(title: "Action needed", message: "Please grant camera permission to use barcode scanner", preferredStyle: .alert)
+                            let langCode = Locale.current.languageCode;
+                            var message = "We need to use the camera to scan codes.";
```

Comment:
It is a bit unnecessary to assign the English strings first if the language is not English I think. Also this wouldn't scale if we introduce another language. 
Maybe replace it with some kind of `switch` statement?

At some point, we should also extract the translations to their own component which handles the language selection and simply gives you the translation of the requested key in the currently selected language. But I guess we don't need that yet

Reply:
I will replace it with switch statement and use en as default.